src/main/ApplicationImpl.cpp

```diff
@@ -271,9 +271,29 @@ ApplicationImpl::getJsonInfo()
     }
 
     auto& herder = getHerder();
-    info["quorum"] =
-        herder.getJsonQuorumInfo(getConfig().NODE_SEED.getPublicKey(), true,
-                                 false, herder.getCurrentLedgerSeq());
+
+    Json::Value quorumInfo;
```

Comment:
should we instead write the code with `Json::Value &quorumInfo = info["quorum"];`
(this avoids extra copies and simplifies the flow a little)

Reply:
Yeah I agree